camera_axis : 
  base_link0: right
  pos: { x: -0.423547301692246, y: 0.38926934603736263, z: 0.5414408158839721 }
  rot: {Rx: 0.3635081261677261, Ry: 0.5953871510703816, Rz: 0.3866105904831357, Rw: 0.6032480699319346 }

#the params are from link to camera 